ilastik/plugins_default/tracking_contour_export.py

```diff
@@ -29,14 +29,15 @@ def export(self, filename, hypothesesGraph, *, labelImageSlot, **kwargs):
 
         :param filename: string of the FILE where to save the result (different .xml files were)
         :param hypothesesGraph: hytra.core.hypothesesgraph.HypothesesGraph filled with a solution
-        :param labelImageSlot: lazyflow.graph.InputSlot, labeled image slot
-        :param kwargs: dict containing additional context info
+        :param pluginExportContext: instance of ilastik.plugins.PluginExportContext containing:
+            - labelImageSlot: lazyflow.graph.InputSlot, labeled image slot
```

Comment:
i wonder if that's obvious, or if one should say that this is the only _mandatory_ key in the `PluginExportContext`, as `PluginExportContext` contains more!

Reply:
:white_check_mark: